CHEATSHEET.md

```diff
@@ -19,13 +24,56 @@ If you miss something, feel free to open an issue or pull request.
 ## Combinators
 
 ### General naming guideline
-
 * `@` means roughly "at this rate"
+* `*` means roughly "in data-parallel"
+* `|` and `+` both roughly means "in time-parallel"
 * `>` and `-` are in the direction of data flow
 
-### Common combinators in action
+### Common combinators
+
+#### SN composition
+
+##### Sequential composition
+| Operator        | Description                                                                                | Simplified Type Signature                                                |
+|-----------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|
+| `>>>` and `<<<` | Arrow composition                                                                          | `Category cat => cat a b -> cat b c -> cat a c`                          |
+| `>->` and `<-<` | Same as `>>>` and `<<<` but with higher operator precedence                                |                                                                          |
+| `^>>>`          | Precompose a signal network with a pure function                                           | `(a -> b) -> SN m cl b c -> SN m cl a c`                                 |
+| `>>>^`          | Postcompose a signal network with a pure function                                          | `SN m cl a b -> (b -> c) -> SN m cl a c`                                 |
+
+##### Parallel composition
+| Operator   | Clock Type | Input Type | Output Type | Resulting Type                             |
+|------------|------------|------------|-------------|--------------------------------------------|
+| `****`     | Same       | Different  | Different   | `SN m cl (a, c) (b, d)`                    |
+| `++++`     | Different  | Same       | Different   | `SN m (ParClock m clL clR) a (Either b c)` |
+| `\|\|\|\|` | Different  | Same       | Same        | `SN m (ParClock m clL clR) a b`            |
+
+##### Parallel rhine composition
+To compose two `Rhine`s in parallel we need a shared `Schedule` between those two. To share a `Schedule` between two `Rhine`s we need to first construct a `RhineParallelAndSchedule` out of one `Rhine` and one `Schedule`. To do this we can use the constructor `RhineParallelAndSchedule` or the aliases `||@` or `++@`. Then we need to combine that resulting `RhineParallelAndSchedule` with another `Rhine` using `@||` or `@++`
+
+| Operator   | Resulting Type                                     |
+|------------|----------------------------------------------------|
+| `@\|\|`    | `Rhine m (ParallelClock m clL clR) a b`            |
+| `@++`      | `Rhine m (ParallelClock m clL clR) a (Either b c)` |
+
+###### Relation between |||| and @||, ++++ and @++
```

Comment:
Same as the comment above. Longer text and explanation has to go to another place than the cheatsheet. Maybe there could be a file `docs/Operators.md` that explains at length what kinds of operators there are, how they work, and when to use them? As a first approximation, you can simply move these texts there.